src/reanimated2/frameCallback/FrameCallbackRegistryUI.ts

```diff
@@ -48,11 +48,12 @@ export const prepareUIRegistry = runOnUI(() => {
             startTime = timestamp;
             this.frameCallbackStartTime.set(key, timestamp);
           }
+          const elapsedTime = timestamp - startTime;
 
           const frameTimings: FrameTimings = {
             timestamp,
-            timeSinceLastFrame,
-            elapsedTime: timestamp - startTime,
+            elapsedTime,
+            timeSinceLastFrame: elapsedTime === 0 ? null : timeSinceLastFrame,
```

Comment:
We don't want to check `elapsedTime === 0`.

Instead, we want to check `startTime === 0` (before we overwrite this).

BTW Why is the default `startTime` 0 and not `undefined`?